pydantic/_internal/_generate_schema.py

```diff
@@ -1655,8 +1626,8 @@ def _sequence_schema(self, items_type: Any) -> core_schema.CoreSchema:
 
         item_type_schema = self.generate_schema(items_type)
         list_schema = core_schema.list_schema(item_type_schema)
+        json_schema = core_schema.list_schema(self.generate_schema(items_type))
```

Comment:
Surprised we don't need the copy here anymore

Reply:
It was deep copying before here, but instead just regenerate a new instance for the "json schema" side. So we dont need the deep copying traversing the whole object tree. Its more consistent with the rest of the logic then.

Reply:
Just as a sanity check - @Viicos, I think you added that copy - does this seem alright to you?